envoy/BUILD

```diff
@@ -25,28 +30,28 @@ envoy_cc_binary(
 
 envoy_proto_library(
     name = "accesslog_proto",
-    srcs = ["accesslog.proto"],
+    srcs = ["cilium/accesslog.proto"],
 )
 
 # TODO: Replace has_services=1 with a new api_go_grpc_library target after rebasing to use data-plane-api's master.
 api_proto_library(
```

Comment:
They also now have new Bazel targets, which may be worth looking into? `api_go_proto_library` and `api_go_grpc_library`

Reply:
I tried them out but would have needed a lot more time to figure out if we can use them. Decided to not go for Bazel build of protos top go this time around.